mssqlcli/jsonrpc/contracts/__init__.py

```diff
@@ -16,18 +16,15 @@ def execute(self):
         """
             Executes the request.
         """
-        pass
 
     @abc.abstractmethod
     def get_response(self):
         """
             Retrieves expected response.
         """
-        pass
```

Comment:
Looking online it seems like using pass is the standard way of defining empty methods. Why are we having to remove it? What's the rule it fixes?

Reply:
`pass` statements are no longer necessary but in this case I think I'd be ok re-including it and adding it as a rule to ignore. Thoughts @kevcunnane?